src/Internal/Payment/State/InitialState.php

```diff
@@ -120,6 +124,7 @@ public function process( PaymentRequest $request ) {
 	 * on all needed parameters being in place.
 	 *
 	 * @param PaymentRequest $request The request to use.
+	 *
```

Comment:
More RadoCS: We don't need an empty line between parameters and exceptions. Same for the whole PR (if there are other instances).